api/proto/node/node.proto

```diff
@@ -33,6 +33,24 @@ service Retrieval {
 	rpc GetBlobHeader(GetBlobHeaderRequest) returns (GetBlobHeaderReply) {}
 	// Retrieve node info metadata
 	rpc NodeInfo(NodeInfoRequest) returns (NodeInfoReply) {}
+
+	////////////////////////////////////////////////////////////////////////////////////////////////
+	// These RPCs are a work in progress and are not yet fully functional. API may not be stable. //
+	////////////////////////////////////////////////////////////////////////////////////////////////
+
+	// GetChunks retrieves all of the chunks for a blob held by the node. Eventually this will replace RetrieveChunks.
+	rpc GetChunks(common.BlobKey) returns (GetChunksReply) {}
```

Comment:
There's some ambiguity here, since the quorum ID isn't specified in the `BlobKey`. Maybe we should have a `common.BlobKeyAndQuorumID` which wraps the `BlobKey`. 

`RetrieveChunks` can take `BlobKeyandQuorumID`;
`ChunkKey` can contain the `BlobKeyAndQuorumID` and the `ChunkIndex`

Reply:
^^Just one idea

Reply:
I ended up just calling the argument `GetChunksRequest`, which contains a blob key and a quorum ID. Do you think we will end up reusing this type? If so, I can move it to common and give it a more general name.